templates/docker/Dockerfile

```diff
@@ -1,13 +1,21 @@
 FROM jacobleecd/mashr:latest
-WORKDIR /root/
+WORKDIR /root
 
 COPY mashr/ mashr/
 RUN chmod +x mashr/install_gems.sh
 RUN mashr/install_gems.sh
 
+# Copy and make executable embulkScript.sh
+COPY embulkScript.sh /etc/cron.d/embulkScript.sh
```

Comment:
Is it correct to put a `sh` script into the `cront.d` directory? Maybe it is, I don't know, it just seemed a little strange to me...I would think it would go in the `app` or `mashr` directories?

Reply:
That's a good point. It makes sense to only store crontabs within the `cron.d` directory. So then yes we can put it in `/root/mashr/` where the other files—`embulk_config.yml.liquid`, `install_gems.sh`, and `keyfile.json`—are.

The VM has the `/app` directory with some our files + the `/app/mashr` directory. The container has `/root/mashr`.